src/nimblepkg/packageinfo.nim

```diff
@@ -530,6 +528,9 @@ proc hash*(x: PackageInfo): Hash =
 proc getNameAndVersion*(pkgInfo: PackageInfo): string =
   &"{pkgInfo.basicInfo.name}@{pkgInfo.basicInfo.version}"
 
+proc isNim*(name: string): bool =
+  result = name == "nim" or name == "nimrod"
```

Comment:
Nim hasn't been known by that name in long enough to exclude it IMO

Reply:
Ok, will do. I will rename a few functions using nimrod in their name as well.

Reply:
> Nim hasn't been known by that name in long enough to exclude it IMO

I reverted that part - nimrod is used in the tests so I don't think that this PR is the place to go and fix them.